raiden-dapp/src/views/Home.vue

```diff
@@ -1,26 +1,141 @@
 <template>
-  <no-tokens v-if="!tokens.length" />
+  <v-container v-if="inaccessible" fluid class="home">
+    <v-row no-gutters>
+      <v-col cols="12">
+        <div class="home__logo-container">
+          <v-img
+            :src="require('../assets/logo.svg')"
+            aspect-ratio="1"
+            class="home__logo-container__logo"
```

Comment:
I think the [BEM naming convention](http://getbem.com/naming) is to use `block__element--modifier` so `home__logo-container--logo` might be a better fit

Reply:
Yes but `logo` is an element also right? modifier would be something like `--active` or `--disabled`

Reply:
I need to read up on these because I always confuse this BEM thingy. :)